portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+ * Class representing a comment that t users create.
+ *
+ * <p>Note: The private variables in this class are converted into JSON.
+ */
+public class Comment {
```

Comment:
Do you need any getters for these variables?

Reply:
At the current moment I do not. Should I add some anyway?

Reply:
Add a TODO.